src/components/migrationToV3.stories.mdx

```diff
@@ -85,6 +92,216 @@ const MyApp = () => {
+If your code looks like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+
+const MyApp = () => {
+  // MyVideoDownlinkBandwidthPolicy is a custom class implementing VideoDownlinkBandwidthPolicy interface from amazon-chime-sdk-js.
+  const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
+  const meetingConfig = {
+    videoDownlinkBandwidthPolicy,
+  };
+
+  const meetingManager = useMeetingManager();
+  await meetingManger.join({
+    data.Meeting,
+    data.Attendee,
+  });
+
+  return (
+  <MeetingProvider {...meetingConfig}>
+    <MyApp />
+  </MeetingProvider>
+  );
+};
+```
+
+change it to like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  // MyVideoDownlinkBandwidthPolicy is a custom class implementing VideoDownlinkBandwidthPolicy interface from amazon-chime-sdk-js.
+  const videoDownlinkBandwidthPolicy = new MyVideoDownlinkBandwidthPolicy();
+
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  meetingSessionConfiguration.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
+
+  const meetingManager = useMeetingManager();
+  await meetingManager.join(
+    meetingSessionConfiguration,
+  );
+
+  return(
+    <MeetingProvider>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+##### simulcastEnabled
+
+One thing to note is that the `MeetingProvider` props might not have the same mapping with `MeetingSessionConfiguration` properties.
+For example, `simulcastEnabled` in `MeetingProvider` and the property in `MeetingSessionConfiguration` is different. The following code example should help remove any confusion.
+
+If your code looks like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+
+const MyApp = () => {
+  const meetingConfig = {
+    simulcastEnabled: false,
+  };
+
+  const meetingManager = useMeetingManager();
+  await meetingManger.join({
+    data.Meeting,
+    data.Attendee,
+  });
+
+  return(
+    <MeetingProvider {...meetingConfig}>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+change it to like this:
+```jsx
+import { useMeetingManager } from 'amazon-chime-sdk-component-library-react';
+import { MeetingSessionConfiguration } from 'amazon-chime-sdk-js';
+
+const MyApp = () => {
+  const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+  meetingSessionConfiguration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = false;
+
+  const meetingManager = useMeetingManager();
+  await meetingManager.join(
+    meetingSessionConfiguration,
+  );
+
+  return(
+    <MeetingProvider>
+      <MyApp />
+    </MeetingProvider>
+  );
+};
+```
+
+##### enableWebAudio
```

Comment:
Can we merge "#### enableWebAudio" section and "##### VideoDownlinkBandwidthPolicy"? Feel like they are the same thing and the two examples are redundant.

You could use a single example to compare the way to config all below options in V2 and V3.

```
- `MeetingSessionConfiguration` props:
  - `VideoDownlinkBandwidthPolicy`
  - `VideoUplinkBandwidthPolicy`
  - `simulcastEnabled`
  - `reconnectTimeoutMs`
  - `keepLastFrameWhenPaused`
- Other props:
  - `logLevel`
  - `postLogConfig`
  - `logger`
  - `enableWebAudio`
  - `activeSpeakerPolicy`
 ```